cupy/core/_routines_statistics.pxd

```diff
@@ -0,0 +1,10 @@
+from cupy.core.core cimport ndarray
+
+
+cdef ndarray _ndarray_max(ndarray self, axis, out, dtype, keepdims)
+cdef ndarray _ndarray_min(ndarray self, axis, out, dtype, keepdims)
+cdef ndarray _ndarray_argmax(ndarray self, axis, out, dtype, keepdims)
```

Comment:
`argmax` and `argmin` are "sorting" routines according to NumPy's categorization, but it's far more convenient to implement it in `_routines_statistics` because they share the internal implementation in common with `max` and `min`.